client/src/components/row/Row.css

```diff
@@ -0,0 +1,32 @@
+.proposal-box {
+    border-radius: 10px;
+    border-width: 1.75px;
+    margin: 0.5em;
+    padding: 1em;
+    background-color: #F3F3F3;
+    border-color: #CECECE;
+    border-style: solid;
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    justify-content: space-between;
+    min-height: 2.5em;
+}
+
+.proposal-title {
+    flex-direction: row;
+    font-family: 'Lexend Deca', sans-serif;
+    color: black;
+    font-size: max(2vw, 1.5vw);
```

Comment:
It's good convention to keep font-size constant in terms of EM rather than dependent on VH